### ISSUE TITLE
The state of the cursor should be remembered if you close accessible view while dynamically streaming

### ISSUE THREAD

**Issue by accesswatch** [microsoft/vscode#293257]
Type: <b>Bug</b>

1. Open Github Chat.
2. Start a chat.
3. Open accessible view.
4. Arrow down a few lines in the text.
5. Press the escape key.
6. Press alt+f2.

Current Behavior:

The cursor is at the top of the text.

Expected Behavior:

The cursor position should be remembered.

VS Code version: Code - Insiders 1.110.0-insider (5bfd50175ce5d5c0a152f23ca9e0875fe26ba107, 2026-02-05T17:07:59.738Z)
OS version: Windows_NT x64 10.0.26200
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 Microsoft Surface (R) Edition (12 x 2196)|
|GPU Status|2d_canvas: enabled<br>GPU0: VENDOR= 0x1002, DEVICE=0x1636 [AMD Radeon(TM) Graphics], DRIVER_VENDOR=AMD, DRIVER_VERSION=27.20.14078.2007 *ACTIVE*<br>GPU1: VENDOR= 0x1414, DEVICE=0x008c [Microsoft Basic Render Driver], DRIVER_VERSION=10.0.26100.7309<br>Machine model name: <br>Machine model version: <br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>trees_in_viz: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|7.45GB (0.90GB free)|
|Process Argv|--crash-reporter-id 020c59d9-47ef-412b-b2d5-3852619a9c60|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (56)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|14.0.0
vscode-intelephense-client|bme|1.16.5
npm-intellisense|chr|1.4.5
vscode-markdownlint|Dav|0.61.1
vscode-eslint|dba|3.0.21
vscode-faker|dee|2.0.0
vscode-axe-linter|deq|4.11.0
githistory|don|0.6.20
prettier-vscode|esb|12.3.0
vscode-firefox-debug|fir|2.15.0
auto-close-tag|for|0.5.15
auto-rename-tag|for|0.1.10
codespaces|Git|1.18.5
copilot-chat|Git|0.38.2026020506
remotehub|Git|0.65.2026012801
vscode-github-actions|git|0.31.0
vscode-pull-request-github|Git|0.128.0
azure-dev|ms-|0.10.0
vscode-azure-github-copilot|ms-|1.0.153
vscode-azure-mcp-server|ms-|1.0.1
vscode-azureappservice|ms-|0.26.4
vscode-azurecontainerapps|ms-|0.10.0
vscode-azurefunctions|ms-|1.20.3
vscode-azureresourcegroups|ms-|0.11.8
vscode-azurestaticwebapps|ms-|0.13.2
vscode-azurestorage|ms-|0.17.1
vscode-azurevirtualmachines|ms-|0.6.10
vscode-containers|ms-|2.4.1
vscode-cosmosdb|ms-|0.30.2
vscode-docker|ms-|2.0.0
vscode-edge-devtools|ms-|2.1.10
data-workspace-vscode|ms-|0.6.3
mssql|ms-|1.39.0
sql-bindings-vscode|ms-|0.4.1
sql-database-projects-vscode|ms-|1.5.6
vscode-pgsql|ms-|1.16.0
playwright|ms-|1.1.17
black-formatter|ms-|2025.3.11831009
debugpy|ms-|2025.19.2026012701
flake8|ms-|2025.3.13521015
mypy-type-checker|ms-|2025.3.13521012
python|ms-|2026.1.2026012801
vscode-pylance|ms-|2025.12.102
vscode-python-envs|ms-|1.17.10361920
remote-containers|ms-|0.443.0
remote-wsl|ms-|0.104.3
azure-repos|ms-|0.41.2026012801
powershell|ms-|2025.4.0
remote-repositories|ms-|0.43.2026012801
vscode-node-azure-pack|ms-|1.8.0
vscode-speech|ms-|0.16.0
windows-ai-studio|ms-|0.30.0
chatgpt|ope|0.5.71
code-spell-checker|str|4.5.3
vscode-ai-foundry|Tea|0.14.0
context7-mcp|Ups|1.0.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
pythonvspyt551:31249597
nativeloc1:31118317
dwcopilot:31158714
dwoutputs:31242946
copilot_t_ci:31333650
g012b348:31231168
pythonrdcb7:31268811
pythonpcpt1:31399616
6518g693:31302842
6abeh943:31336334
envsactivate1:31349248
editstats-enabled:31346256
cloudbuttont:31366566
3efgi100_wstrepl:31403338
839jf696:31457053
use-responses-api:31390341
je187915:31401257
cp_cls_t_966_ss:31454198
chatvisible:31417128
inlinechat_v2_hd992725:31445440
4je02754:31455664
ge8j1254_inline_auto_hint_haiku:31426887
a5gib710:31434435
38bie571_auto:31457539
7a04d226_do_not_restore_last_panel_session:31429685
722j7530_gpt_5_2_codex:31457562
406hc587_ask_agent:31428392
preserve_tokens:31444547
cp_cls_t_1081:31454832
copilot-nes-oct-trt:31432609
ia-use-proxy-models-svc:31446143
nes-slash-models-on:31441832
a43f0574a:31442822
e9c30283:31453065
c0ci8992_validated:31449878
diag_5_5:31448842
nes-trig-ctrl:31449243
idci7584:31454084
edit_mode_hidden:31453871
nes-ftch-ctrl:31454643
nes_chat_context_disabled:31451402
showstats:31452402
e3e4d672:31454087
864ei723_large_tool_results_to_disk:31455802
showingstats:31457201
pp-44cc9697:31455969
nes-fast-add:31454624
i54ji102:31455230
55364912:31455227
7ef87755:31455235
nes-extended-on:31455475

```

</details>
